feat: Add ASC device filtering via snap options (#874)
edgexfoundry/edgex-docs: docs_src/getting-started/Ch-GettingStartedSnapUsers.md

        └── configuration.toml
```

> ??? tip "Filtering devices using snap options"

Reviewer: ```suggestion
??? example "Filtering devices using snap options"
```

>     To filter out events from specific devices/profiles/sources/resources, the snap provides the following configuration options. 
>     For example, to filter out events that come from `Random-Integer-Device` and `Random-Binary-Device`:

Reviewer: ```suggestion
    App service configurable provides various event filtering options.
    For example, to [filter by device names](../../microservices/application/AppServiceConfigurable/#filterbydevicename) `Random-Integer-Device` and `Random-Binary-Device` using snap options:
```
Please verify that the relative path is correct.

    ```
    snap set edgex-app-service-configurable config.writable-pipeline-executionorder="FilterByDeviceName, SetResponseData"
    snap set edgex-app-service-configurable config.writable-pipeline-functions-filterbydevicename-parameters-devicenames="Random-Integer-Device, Random-Binary-Device"
    snap set edgex-app-service-configurable config.writable-pipeline-functions-filterbydevicename-parameters-filterout=true
    ```

>     This is particularly useful when seeding the snap from a [Gadget](https://snapcraft.io/docs/gadget-snap) on an [Ubuntu Core](https://ubuntu.com/core) system.

Reviewer: ```suggestion
```
Not true. This is useful also when installing and configuring the snaps elsewhere.
